Update changelog for 0.15.1 (#9117)
learningequality/kolibri: CHANGELOG.md

List of the most important changes for each release.

## 0.15.1

### Overview of new features
The goals of this release were to fix a bug preventing proper syncing of an individual user's data across multiple devices and to made some small frontend improvements

#### Added

>   - Depreciation warnings for Python 3.4 and 3.5

Reviewer: ```suggestion
  - Deprecation warnings for Python 3.4 and 3.5
```

  - Added auto-alignment property for text display in cards, based on the language

>   - Allow untranslated headers in csv + correct serialization into json

Reviewer: ```suggestion
  - Allow untranslated headers in csv imports and correct serialization into json
```

#### Changed

>   - Several morango bumps, now on latest version v0.6.8

Reviewer: ```suggestion
  - Updated morango to v0.6.8 to support syncing fixes
```

  - Bump zeroconf for fix to properly trigger service update events

>   - Bump KDS version to latest version v1.3.0

Reviewer: ```suggestion
  - Bump KDS version to v1.3.0
```

'latest version' will be false in the near future, so either remove or clarify to 'current latest version'

>   - Updated to latest from Crowdin

Reviewer: ```suggestion
  - Updated translations to support minor translation fixes
```

  - Updated gherkin scenarios for new features

>   - Content API: Change default ordering to "lft"

Reviewer: ```suggestion
  - Content API: Change default ordering to combination of "lft" and "id"
```

#### Fixed

  - Keyboard accessibility/tab navigation focusing for searching and filtering

>   - Allow for scrolling in side panel, and have side panel take up full height of page when 0 results

Reviewer: ```suggestion
  - Allow for scrolling in side panel, and have side panel always take up full height of page even with 0 results
```

  - Small UI improvements including focus ring spacing, button alignment

>   - Perseus to only show hints when allowHints or interactive is true

Reviewer: ```suggestion
  - Hide hints column in Perseus renderer when it could not be displayed to improve display on smaller screens
```

>   - Handle no xAPI statements existing

Reviewer: ```suggestion
  - Handle no xAPI statements existing when calculating H5P and HTML5 progress
```

  - Don't package core node_modules dir

>   - Refactoring card components

Reviewer: ```suggestion
  - Refactor card components for consistency and comprehensibility
```

>   - Update tech debt around KDS theming colors

Reviewer: ```suggestion
  - Address tech debt around KDS theming colors
```
